src/metric_processor.rs

```diff
@@ -106,6 +110,10 @@ mod test {
                      buckets.timer_data().get("some.timer.stddev").unwrap());
         assert_float("23.400",
                      buckets.timer_data().get("some.timer.upper_90").unwrap());
+        assert_float("23.400",
```

Comment:
The values are the same because there's only 4 metric values in buckets. Should I add more?

Reply:
That would add slightly more confidence, wouldn't it?

Reply:
A little. But it means that for me to get a unique 99th I need to generate a list of 100 numbers, which seems overkill.

Reply:
That's true as well. Given that the intention is to move to prometheus, I'd say I'm ok. Besides that, you're not writing code to calculate percentiles. So in an essence you'd be testing code that you didn't write.